levelObjects/Actions.md

```diff
@@ -343,17 +343,27 @@ Control values:
 This change has no parameters (all 12 bytes 0x00) and directly returns to the main menu.
 
 
-#### Change State Type 7: Unknown
-
-    0000  int32      Object index 1
-    0004  int32      Object index 2
-    0008  int32      Unknown
-
-> This entry is only used on level R and Alpha grove to define areas between the two objects. Purpose and effect unknown.
+#### Change State Type 7: Rotate Yaw
```

Comment:
It turns out, it's not just yaw, but rotate object in general. A separate field determines the rotation axis.